iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/writer/HiveIcebergDeleteWriter.java

```diff
@@ -41,21 +41,31 @@
 class HiveIcebergDeleteWriter extends HiveIcebergWriterBase {
 
   private final GenericRecord rowDataTemplate;
+  private final boolean isWriteDeleteRow;
```

Comment:
minor: could we use something like `skipOriginalRow`?

Reply:
@ayushtkn, I like the variant you've used in a tests:`skipRowData`, that seems even better, up to you if you want to change it here.